### ISSUE TITLE
New update suggestion by jimonlinux for https://itsfoss.com/fix-sound-ubuntu-1304-quick-tip/

### ISSUE THREAD

**Issue by abhishekpc** [itsfoss/feedback#96]
I am using Linux Leap 15.5.
I built a desktop computer on an Asus tuf b760-plus wifi board..
I copied files from the previous board that was working fine but after everything was set up I could not get the sound to work.  After checking many times from bios to software and even buying an asus sound card still couldn't get the sound to work. 
One of the options in the article suggested checking hidden files which jogged my memory, this desktop permissions were set so that root was User/owner and Group was Trusted to could make changes. I changed permissions so that root was replaced with the desktop owner.
To sum it up, add to your list of things to check: 
"CHECK PERMISSIONS"
Regards, from someone who should know better,
jimonlinux
